### PR TITLE
Update help documentation for minimum inclusive version range

### LINKED ISSUE #120: Find-PSResource Minimum Inclusive Version is impossible
Specifying a "bare" version number e.g. 2.1.0 returns that exact version, when the nuget version range spec says it should return the latest version with that as the minimum **inclusive** version.
image

`find-psresource powershellget -version 2.1.0`
This should return 2.2.3, but returns 2.1.0

I can understand preserving this behavior to stay compatible and familiar with Find-Module, however as a result there is no way to do a direct inclusive minimum search and you have to do it sort-of indirectly e.g. `(2.999.999, 999.999.999]`

I recommend adding a `-MinimumVersion` parameter that would be inclusive minimum and familiar to PSGetv2 users

### PR DESCRIPTION
**Pull request by anamnavi**
# PR Summary

NuGet version/version range documentation lists supported version inputs, however minimum inclusive version range for NuGet isn't supported for PowerShellGet. The suggested minimum inclusive version range will be considered as a required version. I modified the help docs to include this information and contain the alternative way of providing the minimum inclusive version.

## PR Context

Resolves #120  

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [ ] None
    - **OR**
    - [ ] [Documentation needed]()
        - [ ] Issue filed: 
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [ ] [Documentation needed]()
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [ ] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary.

### CODE REVIEW

**Review by joeyaiello — APPROVED** [review]
LGTM, but it would be good to get a take from @chasewilson and/or @sdwheeler

**Review by alerickson — APPROVED** [review]

File: help/Get-InstalledPSResource.md [hunk 1]
```diff
@@ -89,9 +89,16 @@ Accept wildcard characters: False
...
+Specifies the version of the resource to be returned.
+Can be an exact version or a version range, using the NuGet versioning syntax.
+
+Expected NuGet Version/Version Range format is documented here: https://docs.microsoft.com/en-us/nuget/concepts/package-versioning#version-ranges.
+
+All but the "minimum inclusive version" listed in the NuGet Version/Version Range documentation above are
+supported. The NuGet version documentation suggested minimum inclusive version will be considered as a required version instead in PowerShellGet.
+So inputting "1.0.0.0" as the version won't yield versions 1.0.0.0 and higher (minimum inclusive range)
+but instead will consider that as the required version and yield version 1.0.0.0 only (required version).
+To use the minimum inclusive range, provide "[1.0.0.0, ]" as the version range.
```
Comment (sdwheeler) [inline help/Get-InstalledPSResource.md]:
```suggestion
Specifies the version of the resource to be returned. The value can be an exact version or a version
range using the NuGet versioning syntax.

For more information about NuGet version ranges, see [Package versioning](/nuget/concepts/package-versioning#version-ranges)

PowerShellGet supports all but the _minimum inclusive version_ listed in the NuGet version range
documentation. So inputting "1.0.0.0" as the version doesn't yield versions 1.0.0.0 and higher
(minimum inclusive range). Instead, the values is considered as the required version and yields 
version 1.0.0.0 only (required version). To use the minimum inclusive range, provide `[1.0.0.0, ]` as 
the version range.
```

File: help/Save-PSResource.md [hunk 2]
```diff
@@ -66,9 +66,16 @@ Accept wildcard characters: False
...
+Specifies the version of the resource to be saved.
+Can be an exact version or a version range, using the NuGet versioning syntax.
+
+Expected NuGet Version/Version Range format is documented here: https://docs.microsoft.com/en-us/nuget/concepts/package-versioning#version-ranges.
+
+All but the "minimum inclusive version" listed in the NuGet Version/Version Range documentation above are
+supported. The NuGet version documentation suggested minimum inclusive version will be considered as a required version instead in PowerShellGet.
+So inputting "1.0.0.0" as the version won't yield versions 1.0.0.0 and higher (minimum inclusive range)
+but instead will consider that as the required version and yield version 1.0.0.0 only (required version).
+To use the minimum inclusive range, provide "[1.0.0.0, ]" as the version range.
```
Comment (sdwheeler) [inline help/Save-PSResource.md]:
```suggestion
Specifies the version of the resource to be returned. The value can be an exact version or a version
range using the NuGet versioning syntax.

For more information about NuGet version ranges, see [Package versioning](/nuget/concepts/package-versioning#version-ranges)

PowerShellGet supports all but the _minimum inclusive version_ listed in the NuGet version range
documentation. So inputting "1.0.0.0" as the version doesn't yield versions 1.0.0.0 and higher
(minimum inclusive range). Instead, the values is considered as the required version and yields 
version 1.0.0.0 only (required version). To use the minimum inclusive range, provide `[1.0.0.0, ]` as 
the version range.
```

File: help/Update-PSResource.md [hunk 3]
```diff
@@ -169,8 +169,15 @@ Accept wildcard characters: False
...
-Expected version/version range format is documented here: https://docs.microsoft.com/en-us/nuget/concepts/package-versioning#version-ranges
+Specifies the version of the resource to be updated to.
+Can be an exact version or a version range, using the NuGet versioning syntax.
+
+Expected NuGet Version/Version Range format is documented here: https://docs.microsoft.com/en-us/nuget/concepts/package-versioning#version-ranges.
+
+All but the "minimum inclusive version" listed in the NuGet Version/Version Range documentation above are
+supported. The NuGet version documentation suggested minimum inclusive version will be considered as a required version instead in PowerShellGet.
+So inputting "1.0.0.0" as the version won't yield versions 1.0.0.0 and higher (minimum inclusive range)
+but instead will consider that as the required version and yield version 1.0.0.0 only (required version). To use the minimum inclusive range, provide "[1.0.0.0, ]" as the version range.
```
Comment (sdwheeler) [inline help/Update-PSResource.md]:
```suggestion
Specifies the version of the resource to be returned. The value can be an exact version or a version
range using the NuGet versioning syntax.

For more information about NuGet version ranges, see [Package versioning](/nuget/concepts/package-versioning#version-ranges)

PowerShellGet supports all but the _minimum inclusive version_ listed in the NuGet version range
documentation. So inputting "1.0.0.0" as the version doesn't yield versions 1.0.0.0 and higher
(minimum inclusive range). Instead, the values is considered as the required version and yields 
version 1.0.0.0 only (required version). To use the minimum inclusive range, provide `[1.0.0.0, ]` as 
the version range.
```

**Review by sdwheeler — CHANGES_REQUESTED** [review]
See my suggetions

File: help/Install-PSResource.md [hunk 4]
```diff
@@ -72,9 +72,16 @@ Accept wildcard characters: False
...
+Specifies the version of the resource to be installed.
+Can be an exact version or a version range, using the NuGet versioning syntax.
+
+Expected NuGet Version/Version Range format is documented here: https://docs.microsoft.com/en-us/nuget/concepts/package-versioning#version-ranges.
+
+All but the "minimum inclusive version" listed in the NuGet Version/Version Range documentation above are
+supported. The NuGet version documentation suggested minimum inclusive version will be considered as a required version instead in PowerShellGet.
+So inputting "1.0.0.0" as the version won't yield versions 1.0.0.0 and higher (minimum inclusive range)
+but instead will consider that as the required version and yield version 1.0.0.0 only (required version).
+To use the minimum inclusive range, provide "[1.0.0.0, ]" as the version range.
```
Comment (sdwheeler) [inline help/Install-PSResource.md]:
```suggestion
Specifies the version of the resource to be returned. The value can be an exact version or a version
range using the NuGet versioning syntax.

For more information about NuGet version ranges, see [Package versioning](/nuget/concepts/package-versioning#version-ranges)

PowerShellGet supports all but the _minimum inclusive version_ listed in the NuGet version range
documentation. So inputting "1.0.0.0" as the version doesn't yield versions 1.0.0.0 and higher
(minimum inclusive range). Instead, the values is considered as the required version and yields 
version 1.0.0.0 only (required version). To use the minimum inclusive range, provide `[1.0.0.0, ]` as 
the version range.
```

**Review by sdwheeler — APPROVED** [review]

**Pull request merged**